packages/@react-spectrum/form/stories/Form.stories.tsx

```diff
@@ -88,6 +88,14 @@ storiesOf('Form', module)
     'isQuiet, labelPosition: side',
     () => render({isQuiet: true, labelPosition: 'side'})
   )
+  .add(
+    'isQuiet, labelPosition: side, invalid',
+    () => render({isQuiet: true, labelPosition: 'side', validationState: 'invalid'})
+  )
```

Comment:
I noticed that the quiet textarea has the resize drag handle here, is that intentional?

Reply:
IMO quiet textareas shouldn't be resizable via a drag handle